website/views.py

```diff
@@ -165,10 +165,16 @@ def serialize_log(node_log, auth=None, anonymous=False):
         'params': sanitize.unescape_entities(node_log.params),
         'date': utils.iso8601format(node_log.date),
         'node': node_log.node.serialize(auth) if node_log.node else None,
-        'anonymous': anonymous
+        'anonymous': anonymous,
+        'can_view': can_view(node_log, auth) if node_log.action == NodeLog.FILE_MOVED else True,
     }
 
 
+def can_view(node_log, auth):
```

Comment:
In light of [Sara's comment](https://github.com/CenterForOpenScience/osf.io/pull/5374#issuecomment-205920955) I'm not sure if this is the right approach. (Also, while I haven't verified this, it seems like if you were to request a 'restricted' log from the API, you would still get all the restricted info, plus a flag saying you can't view.)

Instead, consider adding a helper function that, when `<log>.params` are serialized, checks the permissions of the requesting user and does any replacement there, rather than when the logs are rendered. Pseudocode:
```python
def sanitize_params(auth, log):
    if log.action not in UNSAFE_ACTIONS:
        return sanitize.unescape_entities(log.params)
    safe_params = copy.deepcopy(sanitize.unescape_entities(log.params))
    for key in safe_params.keys():
        if key in UNSAFE_KEYS:
            if not check_auth:
                 safe_params[key] = whatever_sanitization_process
    return safe_params
```